modules/ROOT/pages/changelog.adoc

```diff
@@ -4,23 +4,23 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
-== 6.8 - 2023-11-22
+== 6.8.0 - 2023-11-22
 
-== Added
+=== Added
 * CSS files are now also generated as separate JS files to improve bundling of all resources.
 * Added new StylesheetLoader.loadRawCss API that can be used to load CSS into a style element.
 * Added new StylesheetLoader.unloadRawCss API that can be used to unload CSS that was loaded into a style element.
-* Added force_hex_color editor option. Option 'always' converts all RGB & RGBA colours to hex, 'rgb_only' will only convert RGB and not RGBA colours to hex, 'off' won't convert any colours to hex.
+* Added force_hex_color editor option. Option 'always' converts all RGB & RGBA colors to hex, 'rgb_only' will only convert RGB and not RGBA colors to hex, 'off' won't convert any colors to hex.
```

Comment:
I thought American English is fine?

Reply:
As we use `US-en` for tinymce-docs lets go with `color` reference: `In US English, “color” (no “u”) is the correct spelling. In UK English, “colour” (with a “u”) is standard.`